python/test/unit/fem/test_assemble_cppimport.py

```diff
@@ -25,10 +25,11 @@
 def test_eigen_assembly(tempdir):  # noqa: F811
     """Compare assembly into scipy.CSR matrix with PETSc assembly"""
     def compile_eigen_csr_assembler_module():
+        eigen_dir = "/usr/include/eigen3"
```

Comment:
This will break when Eigen is installed elsewhere. Does Eigen support `pkgconfig`?

Reply:
I think so, 
`eigen_dir = dolfinx.pkgconfig.parse("eigen3")["include_dirs"]` provides the eigen include dir.

Reply:
My system doesn't have a pkgconfig file for Eigen, so we your elegantly skip the test if `dolfinx.pkgconfig.parse` doesn't find Eigen.

Reply:
I found `eigen3.pc` - pkg-config path was messed up.